metacluster/include/metacluster/ListTenants.actor.h

```diff
@@ -94,13 +96,15 @@ Future<std::vector<std::pair<TenantName, MetaclusterTenantMapEntry>>> listTenant
 }
 
 ACTOR template <class Transaction>
-Future<std::vector<std::pair<TenantName, MetaclusterTenantMapEntry>>> listTenantMetadataTransaction(Transaction tr,
-                                                                                                    TenantNameRef begin,
-                                                                                                    TenantNameRef end,
-                                                                                                    int limit) {
+Future<std::vector<std::pair<TenantName, MetaclusterTenantMapEntry>>> listTenantMetadataTransaction(
+    Transaction tr,
+    TenantNameRef begin,
+    TenantNameRef end,
+    int limit,
+    TenantGroupName tenantGroup) {
 	std::vector<std::pair<TenantName, int64_t>> matchingTenants = wait(listTenantsTransaction(tr, begin, end, limit));
 	std::vector<std::pair<TenantName, MetaclusterTenantMapEntry>> results =
-	    wait(listTenantMetadataTransaction(tr, matchingTenants));
+	    wait(listTenantMetadataTransaction(tr, matchingTenants, tenantGroup));
```

Comment:
I don't think we need to pass the tenant group into this function. You can get an already filtered list of IDs by calling something like `listTenantGroupTenantsTransaction`.